docs/_documentations/mdt-che-installinfo.md

```diff
@@ -1,62 +1,47 @@
+title: Installing Codewind for Eclipse Che
+description: Installing Codewind for Eclipse Che
+keywords: build, deploy, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops, OpenShift, OKD
 duration: 1 minute
 permalink: mdt-che-installinfo
 type: document
-order: 20
-parent: root
 ---
 
 # Installing Codewind for Eclipse Che
+Install Che to use with Codewind or use Codewind with an existing Che installation.
 
-Installing Codewind for Eclipse Che comprises the following steps:
-- [Installing Codewind for Eclipse Che](#installing-codewind-for-eclipse-che)
-  - [Prerequisites](#prerequisites)
-- [Setting up Che](#setting-up-che)
-  - [Installing Che with chectl](#installing-che-with-chectl)
-    - [Updating an existing Che install](#updating-an-existing-che-install)
-    - [Enabling privileged and root containers to run](#enabling-privileged-and-root-containers-to-run)
-
-## Prerequisites
-1. Ensure PersistentVolumes (PV) are set up and support both `ReadWriteOnce` and `ReadWriteMany` and have minimum 1Gi storage.
+### Prerequisites
+1. Set up PersistentVolumes (PVs) that support both `ReadWriteOnce` (RWO) and `ReadWriteMany` (RWX) access modes and have a minimum of 1 Gi storage.
    - One volume is required for Che, and two volumes are required for each Codewind workspace.
-   - For NFS, set 777 permissions for the exported folders and ownership of `nobody:nogroup`.
-   - Because Codewind uses RWX (ReadWriteMany) volumes to provide persistent storage, you need to use NFS for storage on OpenShift 4.
+   - For Network File System (NFS), set 777 permissions for the exported folders and ownership of `nobody:nogroup`.
+   - Because Codewind uses RWX volumes to provide persistent storage, use NFS for storage on OpenShift 4.
 2. Ensure that the cluster can pull images from `docker.io/eclipse` and `quay.io/eclipse`.
-   - Both Eclipse Che and Eclipse Codewind host their Docker images at those locations. Ensure that your cluster can pull from that registry and does not have `ImagePullPolicies` blocking it from accessing Docker Hub.
-3. Set up the ClusterRole for Codewind. Run `kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.8.0/setup/install_che/codewind-clusterrole.yaml`
-
-# Setting up Che
-
-## Installing Che with chectl
-
-The fastest way to install Eclipse Che for use with Codewind is to use the `chectl` CLI. For instructions on installing the `chectl` CLI tool, see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/).
+   - Both Eclipse Che and Eclipse Codewind host Docker images at those locations. Your cluster needs to pull from that Docker registry, and `ImagePullPolicies` cannot block the cluster from accessing Docker Hub.
```

Comment:
Would it be better to say `Both Eclipse Che and Eclipse Codewind host Docker images at those locations. Many clusters have image policies that control which registries can be used to pull images. Consult your cluster documentation and ensure both registries are permitted according to the cluster image pull policies.`